assignmentTest/splab/Gopistol/invitation/src/main/java/com/splab/invitation/domain/member/service/MemberService.java

```diff
@@ -0,0 +1,60 @@
+package com.splab.invitation.domain.member.service;
+
+import com.splab.invitation.domain.member.entity.Member;
+import com.splab.invitation.domain.member.service.dto.InviteMemberCommand;
+import com.splab.invitation.persistence.member.MemberRepository;
+import java.time.Instant;
+import java.util.concurrent.ConcurrentHashMap;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class MemberService {
+
+  private final EmailService emailService;
+  private final MemberRepository repository;
+  private static final Integer KEY_MAX_LENGTH = 6;
+  private static final Integer SLICE_INDEX = 0;
+  private static final ConcurrentHashMap<String, String> db = new ConcurrentHashMap<>();
+
+  public String create(InviteMemberCommand command) {
+    Member member = command.toEntity();
+    repository.save(member);
+
+    String passKey = generatePassKey();
```

Comment:
초대 코드 정도가 더 좋은 표현인 것 같아요
```suggestion
    String invitationCode = generateInvitationCode();
```